### ISSUE TITLE
Possibility to filter stories with CLI param

### ISSUE THREAD

**Issue by jack-sf** [happo/happo-plugin-storybook#10]
For example, `npm run happo --only Button` would only compile and run stories that have `Button` in their story name.

**Comment by trotzig** [comment]
This is in fact a bug/quirk with the `happo-plugin-storybook` plugin. If you use the default Happo way of defining examples (via `-happo.js` files), you can run with the `--only` flag.

**Comment by jack-sf** [comment]
Yeah, but we could somehow enhance this plugin so it supports this `--only` option as well.
